src/rarkhopper/modals/FormBase.php

```diff
@@ -19,7 +18,7 @@
  */
 abstract class FormBase implements Form{
 	/**
-	 * @param int|bool|array<int, int|string|bool> $rawResponse
+	 * @param int|bool|array<int, int|float|string|bool> $rawResponse
```

Comment:
phpstan + phpstormのみで編集する場合、`int|float|string|bool`を`scalar`に置換可能です
(PHP Intelephense(vscode)非対応)
```suggestion
	 * @param int|bool|array<int, int|float|string|bool> $rawResponse
	 * @phpstan-param int|bool|array<int, scalar> $rawResponse
```
[phpstan/discussions#4507](https://bit.ly/3VQg9rE)

Reply:
vscodeのプラグインでは対応していないものの
公式であるphpstanでは対応されているため修正・コミットしました